src/plugins/workspace/public/utils.ts

```diff
@@ -32,14 +37,14 @@ export const getUseCaseFromFeatureConfig = (featureConfig: string) => {
 
 export const isFeatureIdInsideUseCase = (
   featureId: string,
-  featureConfig: string,
+  useCaseId: string,
   useCases: WorkspaceUseCase[]
 ) => {
-  const useCase = useCases.find(({ id }) => id === getUseCaseFromFeatureConfig(featureConfig));
-  if (useCase) {
-    return useCase.features.includes(featureId);
-  }
-  return false;
+  const availableFeatures =
```

Comment:
I thought we will just bypass all the features in all use case. Why do we change that?

Reply:
Do you mean we don't need to check whether the application id exists in registered use cases? We add skip the feature verification logic outside for all use case workspace.